.github/workflows/ecli-binary.yaml

```diff
@@ -275,10 +275,118 @@ jobs:
+          cp /usr/lib/aarch64-linux-gnu/libz.so /libs
+        dockerRunArgs: |
+          --volume "${PWD}/libbpf:/libbpf"
+          --volume "${PWD}/libs:/libs"
+    - name: cache rust build results
+      uses: Swatinem/rust-cache@v2
+      with:
+        workspaces: |
+          ecli
+    - name: Build ${{matrix.run.title}}
+      run: |
+        rustup target add aarch64-unknown-linux-gnu
+        # Install them, so that libbpf will find their headers
+        sudo apt-get install libelf-dev zlib1g-dev
+        cd ecli/${{matrix.run.src_directory}}
+        RUSTFLAGS="-L $(pwd)/../../libs" cargo build --release --target aarch64-unknown-linux-gnu
+        cd ../..
+        cp ./ecli/target/aarch64-unknown-linux-gnu/release/${{matrix.run.exe_name}} .
+        mkdir data
+        cp ${{matrix.run.exe_name}} data
+        cp .github/scripts/copy-libs.sh data
+    - name: Run ecli and see what libs were loaded by ld
+      uses: uraimo/run-on-arch-action@v2
+      with: 
+        arch: aarch64
+        distro: ubuntu22.04
+        run: |
+          apt-get update
+          apt-get install -y libelf-dev zlib1g-dev
+          echo "Installation done"
+          echo "Working directory $(pwd)"
+          chmod +x /data/${{matrix.run.exe_name}}
+          echo "Permission set done"
+          chmod +x /data/copy-libs.sh
+          /data/copy-libs.sh /data/${{matrix.run.exe_name}} --help
+        dockerRunArgs: |
+          --volume "${PWD}/data:/data"
+        shell: /bin/bash
+    - name: Show what libraries were copied
+      run: |
+        sudo chmod 777 ./data -R
+        sudo chown $(whoami) ./data -R
+        find ./data/libs -type f
+    - name: Prepare runtime file and cargo-appimage tool
+      uses: eunomia-bpf/prepare-cargo-appimage@v6
+      with: 
+        runtime-url: https://github.com/eunomia-bpf/prepare-cargo-appimage/raw/v6/runtime-aarch64
+    - name: Install libfuse for appimagetool
+      run: |
+        sudo apt-get update
+        sudo apt-get install libfuse2
+    - name: Clone and install cargo-appimage with aarch64 launcher
+      run: |
+        git clone https://github.com/eunomia-bpf/cargo-appimage
+        cd cargo-appimage
+        CARGO_TARGET_AARCH64_UNKNOWN_LINUX_GNU_LINKER=aarch64-linux-gnu-gcc RUSTFLAGS="-C target-feature=+crt-static" cargo build --release --target aarch64-unknown-linux-gnu
+    - name: Compose files and pack them
+      run: |
+        mkdir appimage-root
+        cd appimage-root
+        cp -r ../data/libs/* .
+        mkdir -p usr/bin
+        cp ../${{matrix.run.exe_name}} usr/bin/bin
+        touch icon.png
+        cp ../cargo-appimage/target/aarch64-unknown-linux-gnu/release/cargo-appimage-runner AppRun
+        cp ../.github/scripts/cargo-appimage.desktop .
+        ../appimagetool --runtime-file ../runtime . ../${{matrix.run.exe_name}}-aarch64
+    - uses: actions/upload-artifact@v3
+      name: Upload build result
+      with:
+        name: ${{matrix.run.exe_name}}-aarch64
```

Comment:
Will it release ecli or ecli-rs?

Reply:
The executable will be named `ecli-rs-aarch64`

Reply:
Now it's named ecli-arch64, see https://github.com/Officeyutong/eunomia-bpf/releases/tag/v0.0.15 for previewing